agent/ecs_client/model/ecs/api.go

```diff
@@ -26,8 +26,8 @@ const opCreateCluster = "CreateCluster"
 
 // CreateClusterRequest generates a "aws/request.Request" representing the
 // client's request for the CreateCluster operation. The "output" return
-// value will be populated with the request's response once the request complets
-// successfuly.
+// value will be populated with the request's response once the request completes
```

Comment:
Please revert changes to this file.  This probably means we need to regenerate the JSON model from the proper doc model, and then regenerate the client here.

Reply:
+1. both this and https://github.com/aws/amazon-ecs-agent/pull/1156/files#diff-857ccf24765dde4c0cde56a3e67394fc are auto-generated. Can you please revert both of these files?